TF_Service_dotNet/TouchFree_Service/Startup.cs

```diff
@@ -30,12 +30,34 @@ public void ConfigureServices(IServiceCollection services)
             services.AddWebSocketReceiver();
 
             services.AddInteractions();
+
+            services.AddControllers();
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             app.UseWebSockets();
+            app.UseRouting();
+
+            app.UseStaticFiles("/settings");
```

Comment:
I'm pretty sure that we need the `UseCors` because that's the policy with which the files are provided to the browser (If we don't use CORS policies the browser may realise the files are being loaded from the filesystem which they then block). I'll see what I can strip out RE: the others.

Reply:
I take it back, seems like CORS issues aren't present with this hosting method, have stripped this down to just the `UseStaticFiles` without issue.